style.css

```diff
@@ -0,0 +1,94 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&family=Rubik+Gemstones&display=swap');
+*{
+    font-family: 'Rubik Gemstones', cursive;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body{
+    min-height: 100vh;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    background: #000;
+}
+.container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+.container a{
+    position: relative;
+
+    width: 170px;
+    height: 60px;
+    margin: 20px;
+
+
+    display: inline-block;
+
+    background: #fff;
+}
+.container a::after,
+.container a::before{
+    content: "";
+    position: absolute;
+
+    inset: 0;
+
+    background: red;
+    transition: 0.5s;
+}
+.container a span{
+    position: absolute;
+
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    padding: 15px;
+
+    display: inline-block;
+    justify-content: center;
+    align-items: center;
+    font-size: 30px;
+    color: #fff;
+
+    letter-spacing: 2px;
+    text-transform: uppercase;
+    background: #032e0f;
+
+    border: 1px solid #032e0f;
+
+    z-index: 1;
+    overflow: hidden;
+}
+.container a:hover::before{
+    inset: -4px;
+}
+.container a:hover::after{
+    inset: -4px;
+    filter: blur(15px);
+}
+.container a:nth-child(1)::after,
+.container a:nth-child(1)::before{
+    background: linear-gradient(135deg, blue, black, black, red);
+}
+.container a:nth-child(2)::after,
+.container a:nth-child(2)::before{
+    background: linear-gradient(45deg, rgb(255, 196, 0), black, black, rgb(15, 214, 245));
+}
+.container a span::before{
```

Comment:
Please create  Right grouping of  similar property

Reply:
updated it with similar grouping, plz check it out and lemme know if still their is something missing.